samples/calendar/calendar.cpp

```diff
@@ -355,6 +363,10 @@ bool MyApp::OnInit()
     if ( !wxApp::OnInit() )
         return false;
 
+    // Locale affects on the language used in the calendar, and may affect
+    // on the first day of the week.
+    new wxLocale(wxLANGUAGE_DEFAULT);
```

Comment:
This is a memory leak and while it's not exactly critical in a sample, I'd still prefer to avoid showing the wrong example in our code. Could you please just make it a member variable? Then we wouldn't need to allocate it on the heap at all, let alone free it.